nextflow.config

```diff
@@ -23,6 +23,12 @@ prov {
         wrroc {
             file = "${params.outdir}/ro-crate-metadata.json"
             overwrite = true
+            agent {
+                name = "John Doe"
+                orcid = "https://orcid.org/0000-0000-0000-0000"
+            }
```

Comment:
I think we can pull this information from the new `manifest.contributors` config scope now:

https://www.nextflow.io/docs/latest/reference/config.html#config-manifest

Reply:
This new output format for the nf-prov plugin creates RO-Crates that follow [this profile](https://www.researchobject.org/workflow-run-crate/profiles/provenance_run_crate/).
The agent mentioned in the config is not the persons who wrote the workflow, but the person who runs the workflow.